integrations/astra/tests/test_document_store.py

```diff
@@ -172,6 +172,40 @@ def test_delete_documents_more_than_twenty_delete_ids(self, document_store: Astr
         # No Document has been deleted
         assert document_store.count_documents() == 0
 
+    def test_filter_documents_nested_filters(self, document_store, filterable_docs):
+        """
+        Test filter_documents() with nested filters. Two documents from the filterable_docs mixin
+        should be found. If those documents change, this test has to change as well.
+        """
+        filter_criteria = {
+            "operator": "AND",
+            "conditions": [
+                {"field": "meta.page", "operator": ">=", "value": "100"},
```

Comment:
">=" operator is not applicable on strings so this will fail.
```suggestion
                {"field": "meta.page", "operator": "==", "value": "100"},
```

Reply:
Right. Good catch here.